pkg/controller/defaults/resources.go

```diff
@@ -9,7 +9,7 @@ var (
 	// DaemonResources map contains the default resource requirements for the
 	// various OCS daemons
 	DaemonResources = map[string]corev1.ResourceRequirements{
-		"osd": corev1.ResourceRequirements{
```

Comment:
Why are we making these changes?

Reply:
Its a change in `gofmt` that remove unnecessary type declarations

Reply:
Ah, okay.